ksp.html

```diff
@@ -0,0 +1,108 @@
+  <meta name="MobileOptimized" content="width" />
+  <meta name="HandheldFriendly" content="true" />
+  <meta name="apple-mobile-web-app-capable" content="yes" />
+  <link href='https://fonts.googleapis.com/css?family=Chivo:900' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="http://cfgmgmtcamp.eu/stylesheets/stylesheet.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="http://cfgmgmtcamp.eu/stylesheets/pygment_trac.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="http://cfgmgmtcamp.eu/stylesheets/print.css" media="print" />
+  <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.6.4/leaflet.css" />
+  <link rel="shortcut icon" href="http://cfgmgmtcamp.eu/favicon.ico">
+  <!--[if lte IE 8]>
+  <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.6.4/leaflet.ie.css" />
+  <![endif]-->
+  <!--[if lt IE 9]>
+  <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+  <![endif]-->
+  <script src="http://cdn.leafletjs.com/leaflet-0.6.4/leaflet.js"></script>
+  <title>Config Management Camp 2016, Gent</title>
+</head>
+
+<body>
+    <nav>
+      <a href="#" id="menu-icon">&#x2261;</a>
+      <ul>
+      <li><a href="#about">About</a></li>
+      <li><A href="#news">News</a></li>
+      <li><a href="#registration">Registration</a></li>
+      <li><a href="#sponsors">Sponsors</a></li>
+      <li><a href="#route">Location</a></li>
+      <li><a href="#contributions">CFP</a></li>
+      <li><a href="code_of_conduct.html">Code of Conduct</a></li>
+      <li><a href="2014/">2014</a></li>
+      <li><a href="2015/">2015</a></li>
+      </ul>
+    </nav>
+
+  <div id="container">
+    <header>
+      <div class="inner">
+        <h1>#cfgmgmtcamp</h1>
+        <div class="camp-info">
+          <h2><strong>1</strong> and <strong>2</strong> February 2016</h2>
+          <h2>Gent, Belgium </h2>
+        </div>
+      </div>
+    </header>
+
+
+    <div class="inner">
+      <section id="content">
+        <h1>Key Signing Party</h1>
+
+        <p>In tradition with other FOSS events we'll be holding a PGP Key
+        Signing Party (KSP) this year. By holding a KSP we strengthen
+        the Web of Trust (WoT) and allow community members that are
+        likely already in contact with each other to exchange signatures.</p>
+
+        <p>If you've ever been to a KSP before (like the one at FOSDEM) you'll
+        be familiar with the protocol. Here's how this is going to work:</p>
+        <ul>
+          <li>Upload your key to the <a href="https://sks-keyservers.net">GnuPG/sks-keyservers</a> pool of keyservers</li>
+          <li>Send an email to cfgmgmtcampksp@tutanota.com with:
+            <ul>
+              <li>subject: KSP: &lt;name&gt; &lt;0xlongkeyid&gt; &lt;fingerprint&gt;</li>
```

Comment:
Overloading the subject like this doesn't make sense
Make the subject be 'KSP'
and the email contain the output of `gpg --fingerprint mykey`